data/versions.yaml

```diff
@@ -3,10 +3,8 @@
 # They can be referred to with the custom Hugo shortcode, e.g. {{% version "cucumberjvm" %}}
 #
 
-# Note: When updating cucumber-jvm version, please update both the snapshot version and the regular version
-cucumberjvm-snapshot: "5.0.0-SNAPSHOT"
-cucumberjvm: "4.7.4"
-cucumberarchetype: "4.2.6.1"
+cucumberjvm: "5.0.0-RC1"
```

Comment:
should we be documenting / referencing this? We don't for ruby (Use the rc1 package)

Reply:
Good point. I'll merge this branch when v5.0.0 is released and change that.

Reply:
Maybe put WIP to prevent accidental merge?